RegistryMetadata.md

```diff
@@ -2,25 +2,25 @@
 
 ## Introduction
 
-We have 3 data exchange mechanisms as described in the [Architecture Principles](README.md).  In as much as possible, we want to create the same metadata for each method, and expose that metadata through a common query device: The SmartAPI registry.   Below, these methods are abbreviated as S (for SmartAPI), T (for Translator Reasoner API), and G (for Knowledge Graph Exchange).  
+We have 3 data exchange mechanisms as described in the [Architecture Principles](README.md).  In as much as possible, we want to create the same metadata for each method, and expose that metadata through a common query device: The SmartAPI registry.   The SmartAPI contains a metaKG that describes the edges that each service can provide.   Most metadata relates to services that are going to expose information via the metaKG.  low, these methods are abbreviated as S (for SmartAPI), T (for Translator Reasoner API), and G (for Knowledge Graph Exchange).  
 
-This document describes, broadly, the particular metadata that Translator data exchange methods must or should expose. The technical details of how this metadata is specified is located [here]() for SmartAPI-type interfaces, [here]() for Translator Reasoner APIs, and [here]() for Knowledge Graph Exchange files.  
+Additionally, the registry will contain information about services, such as ARAs, that do not expose edges via the metaKG.  ARAs also implement the TRAPI interface, but will not expose the same metadata as KP-related TRAPI interfaces.  These ARA-style interfaces are abbreviated below as A (for ARA).
```

Comment:
So does "T" refer only to KPs now?

Reply:
Yes, but maybe it needs to be more explicit?

Reply:
It makes sense that T refers to KPs **only** - it is cleaner.  We realize that as we now in imPROVE ARA also think about exposing our internal Big KG, SPOKE as a KP to the Translator. But then, we just _act_ as a KP and follow the rules for KPs. Thus, it is cleaner for ARAs to define some part of their functionality as "acting as a KP"  (equivalent to a 'KP component') than trying to fit ARA functionality to TRAPI (or vice versa).